docs/user-docs/latest/settings/configuring-users-permissions-plugin-settings.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Users & Permissions plugin settings - Strapi User Guide
+description: Instructions to configure the Users & Permissions plugin
+---
+
+# Configuring Users & Permissions plugin settings
+
+The Users & Permissions plugin is managed from the *Users & Permissions plugin* settings section, accessible from *General > Settings* in the main navigation of the admin panel. This settings section allows to configure the available providers, email templates and the advanced settings of the plugin. It also allows to define the end-users roles and their related permissions (see [Configuring end-users roles](../users-roles-permissions/configuring-end-users-roles.md)).
+
+## Configuring providers
+
+The Users & Permissions plugin allows to enable and configure providers, for end-users to login via a third-party provider to access the content of a font-end application through the Strapi application API. By default, a list of providers is available including one, "Email", enabled by default for all Strapi applications with the Users & Permissions plugin installed.
```

Comment:
I was wondering whether we should write:
* `front-end`,
* `front end`,
* or `frontend`

I tend to use the last one, but apparently [I was wrong](https://css-tricks.com/poll-results-front-end-front-end/). So thanks for (indirectly) pointing this out, Mégane! 🤓

Reply:
I quite like the rule explained in the article your mentioned in your comment ("front-end" for compound adjectives VS "front end" for nouns) so let's do that!